### PR TITLE
iframe-resizer added to embed LBSD-684

### PR DESCRIPTION
**Pull request by vied12**
to allow dynamic height for iframe depending of the content

### CODE REVIEW

File: server/liveblog/client_modules/client_modules.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ class ClientBlogsResource(BlogsResource):
     schema.update(BlogsResource.schema)
 
 
-class ClientBlogsService(BlogService):
+class ClientBlogsService(BaseService):
```
Comment (nistormihai) [inline server/liveblog/client_modules/client_modules.py]: what all above have to do with iframe-resizer?

File: server/liveblog/themes/themes_assets/angular-base/theme.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
         "//code.angularjs.org/1.4.2/angular-resource.js",
         "//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.3/moment.min.js",
         "vendors/lodash.min.js",
+        "vendors/iframeResizer.contentWindow.min.js",
```
Comment (nistormihai) [inline server/liveblog/themes/themes_assets/angular-base/theme.json]: I think that we should keep angular-base light as possible.

File: server/liveblog/themes/themes_assets/default-theme/main.js [hunk 3]
```diff
@@ -59,6 +59,14 @@
         .then(function() {
             vm.permalinkScroll();
             $interval(retrieveUpdate, UPDATE_EVERY);
+            // listen events from parent
```
Comment (nistormihai) [inline server/liveblog/themes/themes_assets/default-theme/main.js]: can we have a different theme with this iframe functionality?

File: server/liveblog/client_modules/client_modules.py (hunk 1 above)
Comment (vied12) [inline server/liveblog/client_modules/client_modules.py]: It resolves a bug that broke the embed route, because of a new permission check introduced on `devel`. 
Without this, I can't test the embed when I'm based on `devel`.

**Comment by nistormihai** [comment]
In rest everything looks good and a very nice ideea :100: 
PS: we really need to add bower on dependencies, the list became big.

File: server/liveblog/themes/themes_assets/default-theme/main.js (hunk 3 above)
Comment (vied12) [inline server/liveblog/themes/themes_assets/default-theme/main.js]: Yes, I guess.
Do you mean that we should put this listener in `angular-base` ?

File: server/liveblog/client_modules/client_modules.py (hunk 1 above)
Comment (jey07ro) [inline server/liveblog/client_modules/client_modules.py]: please make another commit,
when searching in commit history this will not be clear why it is here.

Comment (nistormihai) [inline server/liveblog/client_modules/client_modules.py]: please make another commit,
when searching in commit history this will not be clear why it is here.
